eva_submission/eload_brokering.py

```diff
@@ -123,11 +123,19 @@ def _collect_brokering_prep_results(self, output_dir):
             os.rename(index_file, output_index_file)
             os.rename(index_file + '.md5', output_index_file + '.md5')
 
+            # .csi index not supported by ENA, so we just save it to be made public later
+            csi_file = os.path.join(output_dir, vcf_file_name + '.csi')
+            output_csi_file = os.path.join(self._get_dir('scratch'), vcf_file_name + '.csi')
```

Comment:
I would prefer if it actually goes to the `18_brokering/ena` folder. 
I know it's weird because we're going to broker it but I don't think we should be storing long lasting file in a `scratch` folder.

Reply:
I agree `scratch` is a weird place, I was a little worried it would be confusing for anybody in the future to have unbrokered files in `brokering` but if you think it's ok I'll do that instead.